rpa-home/modules/ROOT/pages/automation-usecase-legacysystemdataextraction.adoc

```diff
@@ -0,0 +1,107 @@
+# Usecase: Extract Data from a Legacy System
+
+Extracting data from legacy systems is a common use case for RPA. 
+
+This example illustrates the following tasks:
+
+* Copy the contents from a PDF file to a text file.
+* Compare strings.
+* Copy a substring from a text file.
+* Find text on a screen using optical character recognition (OCR).
+* Interact with a screen using mouse actions and key strokes.
+* Use activity parameters to transfer data and to determine which path to follow in the model.
+
+## Example Scenario
+
+Imagine you work as an accountant at Mythical University. Mythical University has ordered a travel package for one of their field excursions. You have received an email and saved the attached order confirmation to your computer.
+
+This is what an order confirmation of your travel agency Wonderlust Travel typically looks like:
+
+[[confirmationletter-image]]
+image::usecase-1-wonderlusttravels-orderconfirmation-pdf.png["Letter containing a logo, headers, text and tables. The order ID is contained in a table in a header in the middle of the page. The label says 'Order ID'.", 75%]
```

Comment:
Is there a way to streamline the alt text into a single sentence fragment?
"Order confirmation letter from Wanderlust Travel, listing the Order ID and Customer ID"

Reply:
I would like to emphasize that the needed information is hard to find at first glance.

Reply:
I see - you could say something like:
"Order confirmation letter from Wanderlust Travel, listing the Order ID and Customer ID in a table in the middle of the page"

I think one suggestion for alt text is to focus on what's important/relevant in the image.

But maybe this is moot if non-sighted people would never use this?

Reply:
It's ultimately up to you. I'm just giving suggestions.